src/content/docs/infrastructure/new-relic-infrastructure/infrastructure-alert-conditions/infrastructure-alerts-add-edit-or-view-host-alert-information.mdx

```diff
@@ -54,52 +54,49 @@ To add an infrastructure alert condition to an alerts policy:
 9. Optional: Set [**Violation time limit** for violations](#violation-time-limit) (this defaults to 24 hours).
 10. Select **Create**.
 
-## Create alert conditions for infrastructure integrations [#integration]
-
-To create an alert condition for your infrastructure integrations, use any of these options:
-
-<table>
-  <thead>
-    <tr>
-      <th style={{ width: "250px" }}>
-        UI options
-      </th>
-
-      <th>
-        Alert conditions for infrastructure integrations
-      </th>
-    </tr>
-  </thead>
-
-  <tbody>
-    <tr>
-      <td>
-        From **Alerts & AI**
-      </td>
-
-      <td>
-        Go to **[one.newrelic.com](http://one.newrelic.com) > Alerts & AI** **> Alerts >** **Alert policies > New alert policy > Create new condition**, then select **Infrastructure** as the product.
+<Callout variant="important">
+  If New Relic hasn't received a Cloud Integration service's attribute in the past 60 minutes, then it won't be selectable for alerting in the UI. You can use the [API](/docs/infrastructure/new-relic-infrastructure/infrastructure-alert-conditions/rest-api-calls-new-relic-infrastructure-alerts/) to create an alert condition for silent attributes.  
```

Comment:
I don't think Cloud Integration needs to be initial caps.
Maybe reword "then it won't be selectable for alerting in the UI" to "then you can't select it in the UI" or "then it won't be an alerting option in the UI"?
Last sentence: I think I know what "silent attributes" means, but it caught me by surprise. Maybe reword to something like:
If New Relic hasn't received a cloud integration service's attribute in the past 60 minutes, we refer to this as a "silent attribute," and it won't be available to use as an alert condition in the UI. In this situation, you can use the API to create alert conditions for silent attributes.

Reply:
Good call!